tdrs-backend/.env

```diff
@@ -0,0 +1,50 @@
+### Local Development Environment
+# Copy this file to `.env` and replace variables as needed
+#
+
+DISABLE_COLLECTSTATIC=1
+
+# Private JWT Key used to generate the client assertion
```

Comment:
We shouldn't commit the backend `.env` file as it contains secret values

Reply:
This commit needs to be removed so it's not available in the repo

Reply:
I'll be rebuilding the history on this.